cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/DatabasesTest.scala

```diff
@@ -211,14 +211,22 @@ class DatabasesTest extends DocumentingTest with QueryStatisticsTestSupport {
         p("In a cluster environment, it may be desirable to change the number of servers used to host a database. " +
           "The number of primary and secondary servers can be specified using the following command. " +
           "For more details on primary and secondary server roles, see <<operations-manual#clustering-introduction-operational, Cluster overview>>")
-        query("ALTER DATABASE `topology-example` SET TOPOLOGY 1 PRIMARY 0 SECONDARIES", ResultAssertions(r => {
+        query("ALTER DATABASE `topology-example` SET TOPOLOGY 3 PRIMARY 0 SECONDARIES", ResultAssertions(r => {
           assertStats(r, systemUpdates = 1)
         })) {
-
-//          statsOnlyResultTable()
+          // execution disabled because `ALTER DATABASE ... SET TOPOLOGY n` fails when running against non-cluster db
         }
         query("SHOW DATABASES yield name, currentPrimariesCount, currentSecondariesCount, requestedPrimariesCount, requestedSecondariesCount", assertDatabasesShown) {
-          resultTable()
+          // Hardcoding results because ALTER DATABASE command doesn't actually execute
+          //resultTable()
+          hardcodedResultTable(Seq("name", "currentPrimariesCount",	"currentSecondariesCount",	"requestedPrimariesCount",	"requestedSecondariesCount"),
+            Seq(
+              ResultRow(Seq("customers", "1", "0", "1", "0")),
```

Comment:
we should have neo4j + some other set-up databases as well here, see output of `SHOW DATABASES` after creating `customers`